@jeremii/todolist/src/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import * as serviceWorker from "./serviceWorker";
+import "./App.css";
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
+
+var items = [];
+items.push({ index: 1, value: "Go jogging", done: true });
+items.push({ index: 2, value: "Buy water", done: true });
+items.push({ index: 3, value: "Call parents", done: false });
+items.push({ index: 4, value: "Cut hair", done: false });
+items.push({ index: 5, value: "Stand up", done: false });
+
+class List extends Component {
+  render() {
+    var items = this.props.items.map((item, index) => {
+      return (
+        <ListItem
+          key={index}
```

Comment:
Using `index` may get you in trouble when filtering or making updates...just be careful.